dashboard/test/ui/test_status.js

```diff
@@ -210,14 +211,26 @@ function renderBrowserProgress(browser, progress) {
 
   // We manipulate the percentages to make them round numbers, adding up to 100,
   // and each category gets at least 1% if its count is greater than zero.
-  let successPercent = Math.max(Math.floor(successCount * 100 / totalCount), successCount > 0 ? 1 : 0);
-  let failurePercent = Math.max(Math.floor(failureCount * 100 / totalCount), failureCount > 0 ? 1 : 0);
-  let pendingPercent = Math.max(Math.floor(pendingCount * 100 / totalCount), pendingCount > 0 ? 1 : 0);
+  let successPercent = Math.max(Math.floor(successCount * 1000 / totalCount)/10, successCount > 0 ? 0.1 : 0);
+  let failurePercent = Math.max(Math.floor(failureCount * 1000 / totalCount)/10, failureCount > 0 ? 0.1 : 0);
+  let pendingPercent = Math.max(Math.floor(pendingCount * 1000 / totalCount)/10, pendingCount > 0 ? 0.1 : 0);
   let leftover = 100 - (successPercent + failurePercent + pendingPercent);
-  if (pendingCount > 0) {
-    pendingPercent += leftover;
-  } else {
-    successCount += leftover;
+  if (leftover > 0) {
```

Comment:
There was a bug before where rounding could add up to a percent greater than 100%, which would result in a fairly arbitrary reduction from the pending percentage, which could display 100% complete before we're actually done.  I've added a single place of precision to the progress bars, and updated the rounding correction to err on the side of incompleteness.

Example bad case before:
```
// Actual counts
successCount = 199
failureCount = 1
pendingCount = 1
// Calculated percentages before accounting for leftover
successPercent = 99
failurePercent = 1
pendingPercent = 1
leftover = -1
```
 By the old logic we would have applied `pendingPercent += leftover` so pendingPercent drops to 0 even though it's nonzero.  The final success/failure/pending result was 99 / 1 / 0. This gave a misleading indication of test completeness:
![screenshot from 2016-06-23 11-25-46](https://cloud.githubusercontent.com/assets/1615761/16315005/47425fb6-3935-11e6-9b38-4068a3124a4a.png)

Here's how the new logic handles this edge case (modified to show edge case in increased granularity):

```
// Actual counts
successCount = 1999
failureCount = 1
pendingCount = 1
// Calculated percentages before accounting for leftover
successPercent = 99.9
failurePercent = 0.1
pendingPercent = 0.1
leftover = -0.1
```

The new logic notes that the leftover is negative and the successes are nonzero, so it deducts from the success percentage first, giving a result of 99.8 / 0.1 / 0.1, correctly indicating that the tests are not done running.